cunumeric/sphinxext/_comparison_config.py

```diff
@@ -0,0 +1,479 @@
+    "set_printoptions",
+    "set_string_function",
+)
+
+IO_ND = ("tofile", "tolist")
+
+MATH = (
+    "absolute",
+    "add",
+    "amax",
+    "amin",
+    "angle",
+    "arccos",
+    "arccosh",
+    "arcsin",
+    "arcsinh",
+    "arctan",
+    "arctan2",
+    "arctanh",
+    "around",
+    "cbrt",
+    "ceil",
+    "clip",
+    "conj",
+    "conjugate",
+    "convolve",
+    "copysign",
+    "cos",
+    "cosh",
+    "cross",
+    "cumprod",
+    "cumsum",
+    "deg2rad",
+    "degrees",
+    "diff",
+    "divide",
+    "divmod",
+    "ediff1d",
+    "exp",
+    "exp2",
+    "expm1",
+    "fabs",
+    "fix",
+    "float_power",
+    "floor_divide",
+    "floor",
+    "fmax",
+    "fmin",
+    "fmod",
+    "frexp",
+    "gcd",
+    "gradient",
+    "heaviside",
+    "hypot",
+    "i0",
+    "imag",
+    "interp",
+    "lcm",
+    "ldexp",
+    "log",
+    "log10",
+    "log1p",
+    "log2",
+    "logaddexp",
+    "logaddexp2",
+    "maximum",
+    "minimum",
+    "mod",
+    "modf",
+    "multiply",
+    "nan_to_num",
+    "nancumprod",
+    "nancumsum",
+    "nanmax",
+    "nanmin",
+    "nanprod",
+    "nansum",
+    "negative",
+    "nextafter",
+    "positive",
+    "power",
+    "prod",
+    "rad2deg",
+    "radians",
+    "real_if_close",
+    "real",
+    "reciprocal",
+    "remainder",
+    "rint",
+    "round_",
+    "sign",
+    "signbit",
+    "sin",
+    "sinc",
+    "sinh",
+    "spacing",
+    "sqrt",
+    "square",
+    "subtract",
+    "sum",
+    "tan",
+    "tanh",
+    "trapz",
+    "true_divide",
+    "trunc",
+    "unwrap",
+)
+
+SEARCHING = (
+    "argmax",
+    "argmin",
+    "argpartition",
+    "argsort",
+    "argwhere",
+    "count_nonzero",
+    "extract",
+    "flatnonzero",
+    "lexsort",
+    "msort",
+    "nanargmax",
+    "nanargmin",
+    "nonzero",
+    "partition",
+    "searchsorted",
+    "sort_complex",
+    "sort",
+    "where",
+)
+
+STATS = (
+    "average",
+    "bincount",
+    "corrcoef",
+    "correlate",
+    "cov",
+    "digitize",
+    "histogram_bin_edges",
+    "histogram",
+    "histogram2d",
+    "histogramdd",
+    "mean",
+    "median",
+    "nanmean",
+    "nanmedian",
+    "nanpercentile",
+    "nanquantile",
+    "nanstd",
+    "nanvar",
+    "percentile",
+    "ptp",
+    "quantile",
+    "std",
+    "var",
+)
+
+MISC = ("kron",)
+
+PACK = ("packbits", "unpackbits")
+
+INDEX = (
+    "c_",
+    "choose",
+    "compress",
+    "diag_indices_from",
+    "diag_indices",
+    "diag",
+    "diagonal",
+    "fill_diagonal",
+    "flatiter",
+    "indices",
+    "ix_",
+    "mask_indices",
+    "ndenumerate",
+    "ndindex",
+    "nditer",
+    "nested_iters",
+    "nonzero",
+    "ogrid",
+    "place",
+    "put_along_axis",
+    "put",
+    "putmask",
+    "r_",
+    "ravel_multi_index",
+    "s_",
+    "select",
+    "take_along_axis",
+    "take",
+    "tril_indices_from",
+    "tril_indices",
+    "triu_indices_from",
+    "triu_indices",
+    "unravel_index",
+    "where",
+)
```

Comment:
In order to whittle down the original lists, I did something like:
```
tuple([x for x in ORIG_INDEX if getattr(np, x, None) is not None])
```
But I want to make sure I am not missing something. Was there a rationale for having things in these lists that did not exist? As an example, the original "index" names list had `('as_strided', 'sliding_window_view')`

Reply:
The way the lists were composed before was copying all the API listed in the documentation. For example, for indexing routines 'as_strided' and 'sliding_window_view' are listed as supported API, however they are implemented in `lib.stride_tricks`. https://numpy.org/doc/stable/reference/arrays.indexing.html
Personally, I am not sure if we should include them or not.

Reply:
I think this is sorted with the updated curated lists.